src/main/java/me/dinowernli/grpc/polyglot/config/ConfigurationLoader.java

```diff
@@ -53,13 +58,58 @@ public static ConfigurationLoader forFile(Path configFile) {
       ConfigurationSet.Builder configSetBuilder = ConfigurationSet.newBuilder();
       String fileContent = Joiner.on('\n').join(Files.readAllLines(configFile));
       JsonFormat.parser().merge(fileContent, configSetBuilder);
+      expandEnvironmentVariables(configSetBuilder);
```

Comment:
I am a bit worried that this replacing might be... overly thorough. Is there a chance we could just do the replacement on a whitelisted set of fields?

Then, a small indicator in the proto file that this happens would be good.